static/app/components/slideOverPanel.tsx

```diff
@@ -93,10 +93,10 @@ const _SlideOverPanel = styled(motion.div, {
 }>`
   position: fixed;
 
-  top: ${space(2)};
-  right: 0;
+  top: ${p => (p.slidePosition === 'left' ? '54px' : space(2))};
```

Comment:
Is there some kind of constant we could use in place of the `54px`? Just trying to see if there's something a bit more semantic